### ISSUE TITLE
Stop using weird unicode characters.

### ISSUE THREAD

**Issue by wildefyr** [Nanoseb/ncTelegram#6]
Main reason is that many terminal bitmap fonts do not have have support for these characters. Specifying a fallback font may be best, but I'd rather see it work like so: if support for a character is found, use that character, if not, use a more common replacement.

I don't have any ideas what we could replace these exotics characters with right now.

**Comment by Nanoseb** [comment]
Yes you are right, I haven't investigate this yet, but knowing if a specific character is handle by the terminal seems quite hard to achieve (for example on my desktop "⤨" works great with terminator but not on urxvt). An other solution would be to add an parameter to the configuration file.
